src/main/java/tech/units/indriya/function/RationalNumber.java

```diff
@@ -123,6 +123,18 @@ public static RationalNumber of(double number) {
 	    return of(decimalValue);
 	}
 
+	/**
+	 * Returns a {@code RationalNumber} that represents the given float
+	 * {@code number}, with an accuracy equivalent to {@code new BigDecimal(Float.toString(number))}.
+	 *
+	 * @param number
+	 */
+	public static RationalNumber of(float number) {
```

Comment:
As mentioned above, breaks the general scheme of first extending floats to double before converting them to BigDecimal or Rational.